tilewe/tournament.py

```diff
@@ -245,19 +260,26 @@ def play(
 
         # helper for getting engine rank summaries
         def get_engine_rankings() -> str:
-            out = f"\n{'Rank':4} {'Name':24} {'Elo':>5} {'Games':>6} {'Score':>10} "
-            out += f"{'Avg Score':>10} {'Wins':>6} {'Win Rate':>9}\n"
-            ranked_engines = sorted(range(N), key=lambda x: -elos[x])
-            for rank, engine in enumerate(ranked_engines):
+            len_name = max(5, min(24, max([len(x.name) for x in self.engines]) + 1))
```

Comment:
I'd like to not have this function duplicated, but it uses so much data it would be really annoying to pass all necessary data into a static version or something. So for now I'm most happy with it duplicated, one here in the running tournament instance and one in the results data class.